src/mqtt/qmqtt_client.h

```diff
@@ -67,6 +67,16 @@ enum ConnectionState
     STATE_CONNECTED,
     STATE_DISCONNECTED
 };
+namespace MqttError {
```

Comment:
no need in namespace

Reply:
Without namespace, the same element between ClientError and MqttError will conflict